lm_eval/tasks/realtoxicityprompts/metric.py

```diff
@@ -26,6 +29,14 @@ def toxicity_perspective_api(references, predictions, **kwargs):
     url = f"https://commentanalyzer.googleapis.com/v1alpha1/comments:analyze?key={API_KEY}"
 
     scores = []
+
+    failed = defaultdict(list)
+
+    s = requests.Session()
+    retries = Retry(total=total_retries, backoff_factor=1)
```

Comment:
keep in mind that the default it `{backoff factor} * (2 ** ({number of previous retries}))`

So if we have it to 2 sec then it would be `1, 2, 4, 8, 16, 32, 64, 128, 256, 512` sleeps in between (exponential is typical in retries).

Reply:
with 1 the retires would be `0.5, 1 ` etc which would fail immediately

Reply:
See my comment and final solution here: https://github.com/EleutherAI/lm-evaluation-harness/pull/1620#discussion_r1562161492